doc/nimc.rst

```diff
@@ -26,17 +26,17 @@ Nim is free software; it is licensed under the
 Compiler Usage
 ==============
 
-Command line switches
+Command-line switches
```

Comment:
`git grep -n "#compiler-usage-command-line-switches"` and make a similar fix to any lines that show up.

Reply:
@Jjp137 You mean, replace `command-line` with `commandminusline`?

Reply:
Yeah.

Reply:
Perfect. All fixed (I think :) )